rllib/optimizers/tests/test_segment_tree.py

```diff
@@ -92,6 +95,20 @@ def test_max_interval_tree(self):
         assert np.isclose(tree.min(2, -1), 4.0)
         assert np.isclose(tree.min(3, 4), 3.0)
 
+    def test_microbenchmark_vs_old_version(self):
```

Comment:
Can you add the outputted numbers as a comment here for future reference?

Reply:
Sure ...